qtop.py

```diff
@@ -1670,20 +1700,28 @@ def scheduler_factory(scheduler, in_out_filenames, config):
                 worker_nodes = scheduling_system.get_worker_nodes()
                 job_ids, user_names, job_states, _ = scheduling_system.get_jobs_info()
                 total_running_jobs, total_queued_jobs, qstatq_lod = scheduling_system.get_queues_info()
-
                 deprecate_old_yaml_files()
 
                 #  MAIN ##################################
                 logging.info('CALCULATION AREA')
-                cluster_dict, NAMED_WNS = calculate_cluster(worker_nodes)
-                workernodes_occupancy, cluster_dict = calculate_wn_occupancy(cluster_dict, user_names, job_states, job_ids)
+                try:
+                    cluster_dict, NAMED_WNS = calculate_cluster(worker_nodes)
+                except EmptySystem:
+                    cluster_dict, NAMED_WNS = dict(), 0
+
+                try:
+                    workernodes_occupancy, cluster_dict, config = calculate_wn_occupancy(cluster_dict, user_names, job_states,
+                                                                                         job_ids, config)
+                except EmptySystem:
+                    workernodes_occupancy, cluster_dict = dict(), dict()
+
                 h_stop = config['h_stop'] if h_stop is None else h_stop
                 v_stop = config['v_stop'] if v_stop is None else v_stop
 
                 display_parts = {
                     'job_accounting_summary': (display_job_accounting_summary, (cluster_dict, total_running_jobs, total_queued_jobs, qstatq_lod)),
                     'workernodes_matrix': (display_wn_occupancy, (workernodes_occupancy, cluster_dict)),
-                    'user_accounts_pool_mappings': (display_user_accounts_pool_mappings, (workernodes_occupancy['account_jobs_table'], workernodes_occupancy['pattern_of_id']))
+                    'user_accounts_pool_mappings': (display_user_accounts_pool_mappings, (workernodes_occupancy,))
```

Comment:
sure, the contents of the parens get unpacked a few lines below, 

```
display_func(*args) if not sections_off[idx] else None
```
If I take out the comma, this is no longer a tuple, which means it will unpack the contents of the dictionary, which is not the intended behaviour.